tableauserverclient/models/workbook_item.py

```diff
@@ -44,6 +45,13 @@ def created_at(self):
     def id(self):
         return self._id
 
+    @property
```

Comment:
Many moons ago, it was because we didn't want to "make any calls the user didn't ask for".

Originally it populated a static list for things like Views in Workbooks; but when we needed paging for nested lists (Users in Groups), I converted everything to an on-demand function call; sometimes this is nice because you can filter or change the paging parameters of nested items. I didn't take it the whole way to making the property take request options and handling all of them to just hidden on-demand calls. Lee had some qualms about mixing requests and models, if I recall. Here, where I bind a secret function to the property was as far as we got :P